web/client/components/map/cesium/DrawMeasureSupport.jsx

```diff
@@ -456,6 +459,15 @@ function DrawMeasureSupport({
                 infoLabelText = infoLabelFormat(convertMeasure(unitOfMeasure, feature.properties.length, 'm'));
             }
             break;
+        case MeasureTypes.LENGTH:
+            if (coordinates.length > 1) {
+                const geodesicDistance = calculateDistance(feature.geometry.coordinates);
+                staticPrimitivesCollection.current.add(createPolylinePrimitive({ ...style?.line, coordinates: [...coordinates], geodesic: true }));
+                segments = addSegmentsLabels(staticLabelsCollection.current, coordinates, MeasureTypes.LENGTH);
```

Comment:
It seems that segments computation is not taking into account the geodesic distance, could you please double check this?

Reply:
you are right, let me check

Reply:
updated